### PR TITLE
Aavechan/wstETH bsc

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

File: src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * @title Onboard wstETH to Aave V3 on BNB Chain
+ * @author Aave Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x752c396a86f1f9b60d3e43b7ed223d9d2f66014e03b6b3eb7ac59e8a169d1fe5
+ * - Discussion: https://governance.aave.com/t/arfc-onboard-wsteth-to-aave-v3-on-bnb-chain/18501
+ */
+contract AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002 is AaveV3PayloadBNB {
+  using SafeERC20 for IERC20;
+
+  address public constant wstETH = 0x26c5e01524d2E6280A48F2c50fF6De7e52E9611C;
+  uint256 public constant wstETH_SEED_AMOUNT = 1e16;
```
Comment (sendra) [inline src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.sol:21]: should be 4e16 ~= 100 usd

**Comment by sendra** [comment]
Saw in this recomendation https://governance.aave.com/t/arfc-onboard-wsteth-to-aave-v3-on-bnb-chain/18501/5 that it should be added to e-mode eth correlated. Is the final decision then to not add for now?

File: src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.t.sol [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers} from 'aave-helpers/src/GovV3Helpers.sol';
+import {AaveV3BNB} from 'aave-address-book/AaveV3BNB.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {IEmissionManager} from 'aave-v3-origin/contracts/rewards/interfaces/IEmissionManager.sol';
+
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/src/ProtocolV3TestBase.sol';
```
Comment (pavelvm5) [inline src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.t.sol:10]:
```suggestion
import {ProtocolV3TestBase} from 'aave-helpers/src/ProtocolV3TestBase.sol';
```

File: src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.sol (hunk 1 above)
Comment (MartinGbz) [reply src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.sol]: done ✅

File: src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.t.sol [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      address(proposal)
+    );
+  }
+
+  function test_collectorHaswstETHFunds() public {
+    GovV3Helpers.executePayload(vm, address(proposal));
+    (address aTokenAddress, , ) = AaveV3BNB.AAVE_PROTOCOL_DATA_PROVIDER.getReserveTokensAddresses(
+      proposal.wstETH()
+    );
+    assertGe(IERC20(aTokenAddress).balanceOf(address(AaveV3BNB.COLLECTOR)), 10 ** 16);
```
Comment (sendra) [inline src/20241002_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241002.t.sol:40]: amount needs to be changed here also

**Comment by sendra** [comment]
Closing as new AIP was created here: https://github.com/bgd-labs/aave-proposals-v3/pull/507

**Pull request closed without merging**